Tests/Integration/Model/OrderTest.php

```diff
@@ -29,15 +31,27 @@ protected function dbData()
                 'columns' => [
                     'oxid',
                     'oxpaymenttype',
+                    'oxdelcountryid',
                     'wdoxidee_orderstate',
                     'wdoxidee_transactionid',
                 ],
                 'rows' => [
-                    ['1', 'wdpaypal', BackendService::TYPE_AUTHORIZED, '1'],
-                    ['2', 'oxidinvoice', BackendService::TYPE_PROCESSING, '2'],
-                    ['3', 'wdcreditcard', BackendService::TYPE_CANCELLED, '1'],
-                    ['4', 'wdcreditcard', BackendService::TYPE_REFUNDED, '2'],
-                    ['5', 'wdpaypal', BackendService::TYPE_PROCESSING, '3'],
+                    ['1', 'wdpaypal', 1, BackendService::TYPE_AUTHORIZED, '1'],
+                    ['2', 'oxidinvoice', 2, BackendService::TYPE_PROCESSING, '2'],
+                    ['3', 'wdcreditcard', 2, BackendService::TYPE_CANCELLED, '1'],
+                    ['4', 'wdcreditcard', 2, BackendService::TYPE_REFUNDED, '2'],
+                    ['5', 'wdpaypal', 2, BackendService::TYPE_PROCESSING, '3'],
+                ],
+            ],
+            [
+                'table' => 'oxcountry',
+                'columns' => [
+                    'oxid',
+                    'oxisoalpha2',
```

Comment:
just to confirm: oxisoalpha2 should not start with **"oxid"**?

Reply:
yes it means ox iso alpha2